blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -120,11 +120,15 @@ export const actions = {
     /**
      * Unconditionally fetch and update the pipelines list.
      * @param config Application configuration.
+     * @param organizationName (optional)
      */
-    fetchPipelines(config) {
+    fetchPipelines(config, organizationName) {
         return (dispatch) => {
-            const url = `${config.getAppURLBase()}` +
-                '/rest/organizations/jenkins/pipelines/';
+            const baseUrl = config.getAppURLBase();
+            const url = !organizationName ?
```

Comment:
Could we ditch the `!` and just swap the following two lines? Then it reads as "if we have organisations, use them in the url like so, otherwise do this"